src/debby/common.nim

```diff
@@ -305,14 +312,14 @@ proc innerSelect*[T: ref object](
     args
   )
 
-macro innerFilter(expression: typed): untyped =
+macro innerFilter[T: ref object](it: var T, db: Db, expression: typed): untyped =
```

Comment:
A generic macro? I don't get why passing db and it is better?

Reply:
My code stores Db as an object, and it's accessed via `app.db`, if the object is also stored with any name other than `db`, it'd also have an error there too

Reply:
I like passing the vars, but I don't like `[T: ref object]` I did it by just passing everything as `typed`

I added a test where you can rename the db.